docs/build.md

```diff
@@ -1,14 +1,13 @@
 # Building the project
 
-1. Install go [here](https://go.dev/doc/install).
+1. Install Golang
 
-For Linux 64-bit:
+On Ubuntu, the easiest way to keep up-to-date with the latest stable version of
+Go is with snap:
 ```bash
-wget https://go.dev/dl/go1.18.linux-amd64.tar.gz
-rm -rf /usr/local/go && tar -C /usr/local -xzf go1.18.linux-amd64.tar.gz
-echo "export PATH=$PATH:/usr/local/go/bin" >> .profile
-source .profile
```

Comment:
can you leave the previous instructions as well? many users are anti-snap :P

Reply:
@noot I'll put them back.  My concern with leaving them in, was that we'll need to update these instructions frequently if the release version is included. The URL we include has the `/usr/loca/go` instructions and it automatically gets updated on every new release.